kubernetes/op-scim-service.yaml

```diff
@@ -8,12 +8,14 @@ spec:
   type: LoadBalancer
   ports:
   - protocol: TCP
-    name: tls
+    name: https
     port: 443
     targetPort: 8443
+# Traffic on :80 is needed to perform the LetsEncrypt certificate challenges, 
```

Comment:
Also used for the status page if we care to mention that.

Reply:
ahh yes I'll add this